lib/config.dart

```diff
@@ -1,10 +1,12 @@
 import 'package:flutter/material.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 
+import 'features/bottom_nav_bar/nav_bar_config.dart';
 import 'features/iparking_chart/utils/range_hour_points.dart';
 
 abstract class MyAppConfig {
   static const title = "ToPwr";
+  static const initialTab = NavBarEnum.home;
```

Comment:
I was thinking if we could create separate config for navigator? And place there all navigation's related variables? What do you think?

Reply:
Yeah, actually it gave me the idea to gather all config files to `/config` directory, cause we have bunch of them somewhere in the code. @mikolaj-jalocha what do you think of that?

Reply:
It would be perfect! I like the idea!